postgres/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Fixed
 
 - `cds build`-relevant files are now correctly packaged into the release. #266
+- The time functions are now correctly casted
```

Comment:
```suggestion
```

no need to manually tweak the changelog, our release automation takes care of that